src/stores/undoStore.ts

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
what about undoStack?
Feels weird using the word store, since we have been exclusively been using the word store as a mobx stores.